src/status_im/config.cljs

```diff
@@ -9,7 +9,7 @@
 
 (defn enabled? [v] (= "1" v))
 
-(goog-define INFURA_TOKEN "")
+(goog-define INFURA_TOKEN "d7aaebc4a3584b5198682ced53801cd3")
```

Comment:
:shushing_face:

Reply:
I will need to remove this infura key and add another. Will address this 👍